### ISSUE TITLE
Sliders disappear from edit scenario pop-up preventing scenario updates

### ISSUE THREAD

**Issue by mabijkerk** [quintel/multi-year-charts#20]
**Description**
For one of the TYNDP scenario the MYC tool shows the following for space heating & hot water:
![Screenshot 2022-09-20 at 12 12 26](https://user-images.githubusercontent.com/67338510/191231589-a5202de0-1a77-4f90-8a23-a51772cd1e66.png)

This matches the shares when I open the 2040 scenario as a single scenario:
![Screenshot 2022-09-20 at 12 13 11](https://user-images.githubusercontent.com/67338510/191231765-bab76a01-f51a-4cd0-94fd-f2d16ab3fcea.png)

However, when I try to edit the scenario in the MYC, suddenly the oil-fired heater and the coal-fired heater disappear:
![Screenshot 2022-09-20 at 12 14 29](https://user-images.githubusercontent.com/67338510/191232084-fbb80600-a4f1-4182-a078-43a656d843c5.png)

**Problem**
When the user tries to update one of the slider values from the MYC tools for the 2040, the update is not processed. I think this is because the slider share group doesn't balance due to the two missing heaters.

**Solution**
@antw could you see if the problem with the disappearing sliders can be fixed?

**Comment by antw** [comment]
This isn't really an MYC issue.

ETModel configures that the coal and oil sliders should only be shown when the dataset has `has_coal_oil_for_heating_built_environment` set to `true`. This scenario uses the NL2019 dataset where this is set to false. I'm assuming these scenarios were created through the API, bypassing ETModel's settings?

I don't think there's much I can do about this in the _short term_. We can have ETModel show sliders like these whenever there is a user value, but that is not trivial; the backend – which decides what sliders can be shown – does not have access to user values. It would be a fair bit of work to change this (presumably to move this responsibility to the front-end which _does_ have access to user values).

**Comment by antw** [comment]
I note that your second screenshot does include the oil and coal sliders. I cannot reproduce this myself; an NL scenario should not have those sliders visible. Are you sure you weren't looking at a scenario for a different dataset?

**Comment by mabijkerk** [comment]
> I note that your second screenshot does include the oil and coal sliders.
Yes, in the second screenshot I open only the 2040 scenario in the ETM, so not in the MYC tool. The scenario isn't an NL scenario, but a FR scenario.
